docs/multicluster/user-guide.md

```diff
@@ -401,9 +401,17 @@ for exporting the cluster's network information to other member clusters
 through the leader cluster, including the cluster's Gateway IP and Service
```

Comment:
Not relevant to this PR, but I do not understand the table at the beginning of this section (table under "Below is a table about communication support for different configurations.").

1. Does it try to explain some configuration does not require a Gateway, when Pod IPs are directly routable across clusters and no support for Service traffic? Is this a supported configuration?
2. What does "MC endpoint" mean? Pod or Service IPs or both are reachable across clusters?
3. Is the first row a supported configuration - no underlay connectivity, no gateway, and no Pod/Service traffic across clusters?

Reply:
Thanks for the explanation. We can consider how to revise the description/table separately. I will read again and think about it.